client/src/components/layouts/default.tsx

```diff
@@ -117,10 +125,13 @@ function DefaultLayout({
   t,
   theme = Themes.Default,
   user,
-  fetchUser
+  fetchUser,
+  loadAllChallengesInfo
 }: DefaultLayoutProps): JSX.Element {
+  const { challengeEdges, certificateNodes } = useGetAllBlockIds();
   useEffect(() => {
     // componentDidMount
+    loadAllChallengesInfo({ challengeEdges, certificateNodes });
```

Comment:
Pretty inconsequential - put this in a `if (isSignedIn)`? We don't need it if they aren't signed in.

Reply:
thanks for the heads up. I does make sense. I will check for side effects in the incoming prs.